src/vm/frames.h

```diff
@@ -2466,17 +2466,22 @@ class ComPrestubMethodFrame : public ComMethodFrame
 //------------------------------------------------------------------------
 class GCFrame
 {
+    void Init(Thread *pThread, OBJECTREF *pObjRefs, UINT numObjRefs, BOOL maybeInterior);
```

Comment:
The can be merged into the constructor. The constructor that does not take `pThread` can forward to constructor that does take `pThread`.

Reply:
Done